packages/react/src/components/Renderer.tsx

```diff
@@ -38,9 +39,19 @@ export const Renderer = (props: RendererProps) => {
     [setMountedElements]
   );
 
-  React.useEffect(() => {
-    const ui: NovuUI = new NovuUI(options);
-    setNovuUI(ui);
+  useEffect(() => {
+    setNovuUI(new NovuUI(optionsRef.current));
+  }, []);
```

Comment:
we should create the NovuUI instance once per component lifecycle

Reply:
Lifecycle as in mount/unmount?

Reply:
yes